### PR TITLE
Feature openapi

### PR DESCRIPTION
**Pull request by ingoboerner**

### CODE REVIEW

**Comment by ingoboerner** [comment]
Don't know why there are these conflicts

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ build/
 .existdb.json
 expath-pkg.xml
 ecocor-latest.xar
+.DS_Store
```
Comment (cmil) [inline .gitignore:6]: Let's remove the .DS_Store

File: api.yaml [hunk 2]
```diff
@@ -1,36 +1,369 @@
...
+  description: About a text
+- name: entities
+  description: About entities
 paths:
   /info:
     get:
       summary: API info
-      description: >-
-        Shows version numbers of the ecocor-api app and the underlying
+      description: Shows version numbers of the dracor-api app and the underlying
```
Comment (cmil) [inline api.yaml:26]: There are still references to dracor. These should be adjusted.

**Review by cmil — COMMENTED** [review]
I realise I hadn't pushed any of my yesterday's changes yet. I just did. @ingoboerner Do you think you could rebase your branch and reconcile your draft with what I had added. Sorry for complicating thing like that.

**Pull request closed without merging**